docs/QUICKSTART.md

```diff
@@ -1,76 +1,195 @@
+    ```
+    
+    ```shell 
+    # create a cluster
+    cat <<EOF | kind create cluster --config -
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    EOF
+    ``` 
+  
+    ```shell
+    # Verify your cluster is running.
+    $ kubectl cluster-info --context kind-kind
+    Kubernetes control plane is running at https://127.0.0.1:40989
+    CoreDNS is running at https://127.0.0.1:40989/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ``` 
+
+2. Deploy Eraser to the Cluster
+
+    ```bash
+    $ kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v0.1.0/deploy/eraser.yaml
+    namespace/eraser-system created
+    customresourcedefinition.apiextensions.k8s.io/imagejobs.eraser.sh created
+    customresourcedefinition.apiextensions.k8s.io/imagelists.eraser.sh created
+    serviceaccount/eraser-controller-manager created
+    role.rbac.authorization.k8s.io/eraser-leader-election-role created
+    clusterrole.rbac.authorization.k8s.io/eraser-manager-role created
+    rolebinding.rbac.authorization.k8s.io/eraser-leader-election-rolebinding created
+    clusterrolebinding.rbac.authorization.k8s.io/eraser-manager-rolebinding created
+    configmap/eraser-manager-config created
+    deployment.apps/eraser-controller-manager created
+    ```
+
+    ```bash
+    $ kubectl get pods -n eraser-system
+    NAME                                         READY   STATUS    RESTARTS   AGE
+    eraser-controller-manager-759c7c8794-q8qqg   1/1     Running   0          22s
+    ```
+
+3. Apply and delete a deployment
+
+    Applying and deleting a deployment will leave unused images on the nodes where it was deployed. For this example, an `ngnix` deployment will be used. In following steps, these images will be used to verify that Eraser is removing the correct images.
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: nginx-deployment
+    spec:
+      selector:
+        matchLabels:
+          app: nginx
+      replicas: 2 
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+          containers:
+          - name: nginx
+            image: nginx:1.14.2
+            ports:
+            - containerPort: 80
+    EOF
+    ```
+
+    ```shell
+    # list pods in the default namespace
+    $ kubectl get po
+    NAME                                READY   STATUS    RESTARTS   AGE
+    nginx-deployment-6595874d85-bvmg4   1/1     Running   0          8s
+    nginx-deployment-6595874d85-h2r9c   1/1     Running   0          7s
+    ```
+
+    ```shell
+    $ kubectl delete deployment nginx-deployment
+    deployment.apps "nginx-deployment" deleted
+    ```
+
+    ```shell
+    # verify the pods have been deleted
+    $ kubectl get po
```

Comment:
suggest using fully qualified commands to help users through the tutorial, i.e. don't abbreviate "pods" to "po"

Reply:
updated in [83eaf48](https://github.com/Azure/eraser/pull/186/commits/83eaf483c7b3f4e7463e0f3a438e51c3ddaed5f9)